src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -99,15 +106,63 @@ export class RelayFeeCalculator {
       this.capitalCostsPercent >= 0 && this.capitalCostsPercent <= 100,
       "capitalCostsPercent must be between 0 and 100 percent"
     );
-    this.capitalCostsConfig = config.capitalCostsConfig || {};
-    for (const token of Object.keys(this.capitalCostsConfig)) {
-      RelayFeeCalculator.validateCapitalCostsConfig(this.capitalCostsConfig[token]);
+    this.capitalCostsConfig = {};
+    for (const token of Object.keys(config.capitalCostsConfig || {})) {
+      this.capitalCostsConfig[token] = RelayFeeCalculator.validateAndTransformCapitalCostsConfigOverride(
+        (config.capitalCostsConfig || {})[token]
+      );
     }
 
     this.logger = logger;
   }
 
-  static validateCapitalCostsConfig(capitalCosts: CapitalCostConfig) {
+  /**
+   * Type guard to check if a config is a CapitalCostConfigOverride or a CapitalCostConfig.
+   * @param config CapitalCostConfig or CapitalCostConfigOverride
+   * @returns true if the config is a CapitalCostConfigOverride, false otherwise.
+   * @private
+   * @dev This is a type guard that is used to check if a config is a CapitalCostConfigOverride or a CapitalCostConfig.
+   * This is needed because the config can be either a CapitalCostConfig or a CapitalCostConfigOverride. If it's a
+   * CapitalCostConfig, then we need to convert it to a CapitalCostConfigOverride with the default config set with no route
+   * overrides.
+   */
+  private static capitalCostConfigIsOverride(
+    config: CapitalCostConfig | CapitalCostConfigOverride
+  ): config is CapitalCostConfigOverride {
+    return (config as CapitalCostConfigOverride).default !== undefined;
```

Comment:
Note: (stylistic, you can do it either way) I think typescript will let you do this without casting if you do this: `"default" in config`.